src/gui/editorwidgets/qgsrelationreferencewidgetwrapper.h

```diff
@@ -63,6 +63,7 @@ class GUI_EXPORT QgsRelationReferenceWidgetWrapper : public QgsEditorWidgetWrapp
     void updateConstraintWidgetStatus() override;
 
   private:
+    void aboutToSave() override;
```

Comment:
make it virtual too?

Reply:
I could. But why?

Reply:
I was thinking at a sub-implementation in Python, but it would require SIP_FORCE and I'm not sure it makes a lot of sense after all.